helix-view/src/input.rs

```diff
@@ -271,6 +273,11 @@ impl From<crossterm::event::Event> for Event {
             crossterm::event::Event::Key(key) => Self::Key(key.into()),
             crossterm::event::Event::Mouse(mouse) => Self::Mouse(mouse.into()),
             crossterm::event::Event::Resize(w, h) => Self::Resize(w, h),
+            crossterm::event::Event::FocusGained => Self::FocusGained,
```

Comment:
Focus events are emitted on Windows regardless of any flags being set in crossterm. Don't want to use `unreachable` like in the Paste arm as a result.